### PR TITLE
Fix: makes dialog players pausable by removing yields

### PR DESCRIPTION
**Pull request by StraToN**
Fixes a blocking issue happening when pausing the game (ESC key) while a say line was ongoing: dialog line was yielding for say_finished signal, thus blocking pause menu

### CODE REVIEW

File: addons/escoria-ui-9verbs/game.tscn [hunk 1]
```diff
@@ -19,6 +19,15 @@ script = ExtResource( 5 )
 main_menu = NodePath("ui/main_menu")
 pause_menu = NodePath("ui/pause_menu")
 
+[node name="dialog_layer" type="CanvasLayer" parent="."]
+
```
Comment (dploeger) [inline addons/escoria-ui-9verbs/game.tscn:23]: You changed the layer here. Originally it was layer 3. Was that intended?

**Review by dploeger — CHANGES_REQUESTED** [review]
Already awesome work. Some remarks.

File: addons/escoria-dialog-simple/esc_dialog_simple.gd [hunk 2]
```diff
@@ -4,7 +4,7 @@ class_name ESCDialogSimple
 
 
 # The currently running player
-var _type_player: Node = null
+var type_player: Node = null
```
[the remark this answers is not in the document]
Comment (StraToN) [reply addons/escoria-dialog-simple/esc_dialog_simple.gd:7]: Oh I didn't see the getter! Thanks!

File: addons/escoria-ui-9verbs/game.tscn (hunk 1 above)
Comment (StraToN) [reply addons/escoria-ui-9verbs/game.tscn:23]: Yes it was intended : the bug was that dialog lines were displayed above the pause menu. The dialog_layer value doesn't care about the layer position in the scene tree: bigger values are above, so I lowered it.

**Review by dploeger — APPROVED** [review]
Great. Thanks! 🎁

**Comment by StraToN** [comment]
Thanks!

**Pull request merged**